stdlib/experimental/usage/usage.flux

```diff
@@ -0,0 +1,48 @@
+package usage
+
+import "experimental/influxdb"
+import "csv"
+import "experimental/json"
+
+// from returns an organization's usage data. The time range to query is
+// bounded by start and stop arguments. Optional orgID, host and token arguments
+// allow cross-org and/or cross-cluster queries. Setting the raw parameter will
+// return raw usage data rather than the downsampled data returned by default.
+//
+// Note that unlike the range function, the stop argument is required here,
+// pending implementation of https://github.com/influxdata/flux/issues/3629.
+from = (start, stop, host="", orgID="{orgID}", token="", raw=false) => {
```

Comment:
what does a default of `{orgID}` do? Seems like it would error if not supplied directly?

Reply:
Organization usage is a cloud-specific concern, so the design of this function began with cloud as the primary focus. The cloud implementation of the `api` function supports transparent interpolation of the organization ID taken off the request authorization into the requested path (that PR can is in draft in the cloud repository). This is intended to allow a customer to access API endpoints that include org or user IDs in the path without having to look up those specific IDs to use this function.

In other words, in the cloud implementation, if the request is authorized as orgID 123, this call

```
influxdb.api(method: "get", path: "/api/v2/orgs/{orgID}")
```

results in an API call to the endpoint "/api/v2/orgs/123" by deriving the caller's organization ID from the authorization on the request context.

Similarly, in the cloud implementation, if no token argument is supplied, the token is taken off the query request itself. The intent of these features is to allow this function to be easy to use for requests to one's local cloud cluster, but _also_ allow for cross cluster requests.

The initial plan was for `api` to be unimplemented outside of cloud, however it occurred to me that while these special auth-based features are cloud-specific, a user could conceivably want use this function from the Flux REPL or an OSS instance to query their cloud usage. Thus I've include a simplified implementation of the cloud function here.

This leads to the awkward situation of the `{orgID}` default not making much sense without that context, however, so I am open to any suggestions or guidance on how best to accomplish my goals here.

Reply:
Thanks for the explanation. I think we could move this abstraction higher in the stack so we do not need a custom builtin. 

Here is the idea we discussed as team. Instead of using sentinel values like `{orgID}` that get replaced we could use an option. 

```
package influxdb

option org = ""
```

Then we inject the value of the current org into that option at runtime. 

Then the `usage` function signature can be:

```
package usage

from = (start, stop, host="", orgID=influxdb.org, token="", raw=false)  
```

Thoughts?